omod/src/main/webapp/resources/scripts/emr.js

```diff
@@ -87,6 +87,22 @@ var emr = (function($) {
             }
         },
 
+        serverValidationErrorMessage: function(response) {
+        	var errorMessage = "";
+        	if (response.data != null && response.data.error != null) {
+        		errorMessage = response.data.error.message;
```

Comment:
Validation error messages don't have sensitive information, it is the other error messages with different codes that can have sensitive data